File: exs/mundo_1/python/030.py
"""
Desafio 030

Problema:  Crie um programa que leia um número inteiro e mostre na tela se ele é PAR ou ÍMPAR.

Resolução do problema:
"""
numero = int(input('Informe um número: '))
print('O número {} é PAR'.format(numero) if numero % 2 == 0 else 'O número {} é ÍMPAR.'.format(numero))

File: exs/mundo_1/rust/030.rs
/*
Desafio 030

Problema: Crie um programa que leia um número inteiro e mostre na tela se ele é
          PAR ou ÍMPAR.
*/

fn main() {
    let mut input = String::new();

    println!("informe um número: ");
    std::io::stdin().read_line(&mut input).unwrap();

    let input = input.trim().parse::<i64>().expect("Não é um número");

    println!(
        "O número {input} é {}",
        if input % 2 == 0 { "par" } else { "impar" }
    );
}
